hapi-fees/src/main/java/com/hedera/services/usage/token/entities/NftEntitySizes.java

```diff
@@ -0,0 +1,38 @@
+package com.hedera.services.usage.token.entities;
+
+/*-
+ * ‌
+ * Hedera Services API Fees
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import static com.hederahashgraph.fee.FeeBuilder.BASIC_ENTITY_ID_SIZE;
+import static com.hederahashgraph.fee.FeeBuilder.BASIC_RICH_INSTANT_SIZE;
+import static com.hederahashgraph.fee.FeeBuilder.LONG_SIZE;
+
+public enum NftEntitySizes {
+	NFT_ENTITY_SIZES;
+
+	public int fixedBytesInNftRepr() {
```

Comment:
Nft metadata should be added here once we update `MerkleUniqueToken` with the `metadata` property
@daniel-danev-limechain